src/PrestaShopBundle/Command/DebugCommand.php

```diff
@@ -0,0 +1,137 @@
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Command;
+
+use Exception;
+use PrestaShop\PrestaShop\Adapter\Debug\DebugMode;
+use PrestaShop\PrestaShop\Adapter\LegacyContextLoader;
+use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
+use PrestaShop\PrestaShop\Core\Domain\Configuration\Command\SwitchDebugModeCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * CLI command for getting and setting debug mode setting
+ */
+class DebugCommand extends Command
+{
+    private const STATUS_OK = 0;
+    private const STATUS_ERROR = 1;
+
+    /**
+     * @var InputInterface
+     */
+    protected $input;
+
+    /**
+     * @var OutputInterface
+     */
+    protected $output;
+
+    /**
+     * @var SymfonyStyle
+     */
+    protected $io;
+
+    /**
+     * @var CommandBusInterface
+     */
+    private $commandBus;
+
+    /**
+     * @var DebugMode
+     */
+    private $debugConfiguration;
+
+    public function __construct(CommandBusInterface $commandBus, DebugMode $debugConfiguration, LegacyContextLoader $legacyContextLoader)
+    {
+        parent::__construct();
+        $this->commandBus = $commandBus;
+        $this->debugConfiguration = $debugConfiguration;
+        $legacyContextLoader->loadGenericContext();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('prestashop:debug')
+            ->setDescription('Get or set debug mode')
+            ->addArgument('value', InputArgument::OPTIONAL, 'Value for debug mode, on/off, true/false, 1/0. If left out will just print the current state')
+        ;
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     *
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        // no new value, just print out the current setting value
+        $inputValue = $this->input->getArgument('value');
+        if (is_null($inputValue)) {
+            $this->io->success(sprintf('Debug mode is: %s', $this->debugConfiguration->isDebugModeEnabled() ? 'ON' : 'OFF'));
+
+            return self::STATUS_OK;
+        }
+
+        // parse incoming to truthy
+        $newValue = filter_var($inputValue, FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE);
+        if (is_null($newValue)) {
```

Comment:
```suggestion
        if ($newValue === null) {
```
You should ban the is_null function which is really bad.

Reply:
maybe the cs-fixer should know this also and automatically do it (or at least complain about it)?